src/main/java/alvioneurope/gsergeev/javagroup/clientlogimporter/ExecutedCommand.java

```diff
@@ -11,5 +15,80 @@
     private String operationId;
     private int responseLength;
     private int execTime;
+    private boolean executedCommand;
+
+    public ExecutedCommand(String executedCommand){
+        this.executedCommand = splitString(executedCommand);
+    }
+
+    public boolean checkString(String string) {
+        if (string == null || string.length() == 0) return false;
+        int i = 1;
+        if (string.charAt(0) == '-') {
+            if (string.length() == 1)
+                return false;
+        }
+        char c;
+        for (; i < string.length(); i++) {
```

Comment:
I didn't understand what do you want to check exactly but I think it isn't good idea use loop in this case